tracer/test/snapshots/OwinWebApi2Tests.CallSite.WithFF.__path=_handler-api_api-ps=true&ts=true_statusCode=200.verified.txt

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
This is expected because the `ts=true` query param issues the request to a message handler that returns a response immediately instead of routing to an api controller.